components/build/shared-resources/shared-resources-components.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: shared-resource-appstudio-push-secret
+rules:
+  - apiGroups:
+      - sharedresource.openshift.io
+    resources:
+      - sharedsecrets
+    resourceNames:
+      - appstudio-push-secret
+    verbs:
+      - use 
+  - verbs:
+      - use
+    apiGroups:
+      - security.openshift.io
+    resources:
+      - securitycontextconstraints
+    resourceNames:
+      - csi-scc
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: shared-resource-appstudio-push-secret
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: shared-resource-appstudio-push-secret
+subjects:
+  - kind: ServiceAccount
```

Comment:
Am I correct @Michkov @sbose78 that the `pipeline` SA in user workspaces will *NOT* need access to this push secret?  Or will they (if not now, eventually) and and assigning this role to them will be done separately, and in another component?

Reply:
yes, this is only for AppStudio components not users. So the CI is able to push into appstudio quay account.